ui/pages/userpreference.js

```diff
@@ -49,7 +50,9 @@ class UserPref extends React.Component {
         <Head>
           <title>Preferences | Meshery</title>
         </Head>
-        <UserPreference anonymousStats={anonymousStats} perfResultStats={perfResultStats}/>
+        <Paper>
```

Comment:
If you can slide this paper up a little to close the gap, and move the two inputs a little closer together vertically so that there is some padding on the top and bottom of the paper, that would be great.

<img width="889" alt="Screen Shot 2020-08-25 at 2 48 58 PM" src="https://user-images.githubusercontent.com/7570704/91220962-967d1a80-e6e2-11ea-924b-9192fd6ad8e5.png">

This is a good update in general. Let's get this merged into `master`.

Reply:
I misremembered what we have today. I take it back, this change is step backwards. ;)

What we have today is better. It does need to bring the two inputs together and add a label: `Analytics and Improvement Program` 

<img width="1284" alt="Screen Shot 2020-08-25 at 2 54 50 PM" src="https://user-images.githubusercontent.com/7570704/91221465-4bafd280-e6e3-11ea-8d54-7e8fa496fe04.png">